packages/argo-checkout/src/components/Banner/Banner.ts

```diff
@@ -0,0 +1,43 @@
+import {createRemoteComponent} from '@remote-ui/core';
+
+/**
+ * Success:
+ * Use rarely, only if you need additional visual confirmation that a
+ * non-standard action has been completed successfully, for example adding an
+ * item to an order as an upsell.
+ *
+ * Warning:
+ * Use to display information that needs attention or that customers need to
+ * take action on. Seeing these banners can be stressful for customers so be
+ * cautious about using them. Does not block progress to next step.
+ *
+ * Critical:
+ * Use to communicate problems that have to be resolved immediately for
+ * customers to complete a task. For example, using a different payment method
+ * if card details couldn't be processed. Seeing these banners can be stressful
+ * for customers so be cautious about using them.
+ */
+type Status = 'info' | 'success' | 'warning' | 'critical';
+
+export interface BannerProps {
+  /**
+   * Banners have an optional title. Use a title to grab the buyers attention
+   * with a short, concise message.
+   */
+  title?: string;
+  /**
+   * What status icon & color scheme to use
```

Comment:
```suggestion
   * Sets the status of the banner.
```

I would prefer not mentioning any visual treatment as this may vary based on theming. We should make sure the status is aligned with its content, not the color or icon we want.